beacon-chain/rpc/prysm/v1alpha1/validator/construct_generic_block_test.go

```diff
@@ -154,7 +155,7 @@ func TestConstructGenericBeaconBlock(t *testing.T) {
 	t.Run("phase0 block", func(t *testing.T) {
 		b, err := blocks.NewSignedBeaconBlock(util.NewBeaconBlock())
 		require.NoError(t, err)
-		result, err := vs.constructGenericBeaconBlock(b, nil)
+		result, err := vs.constructGenericBeaconBlock(b, nil, primitives.ZeroWei)
```

Comment:
note to self: `primitives.ZeroWei` should be a func that returns a new value each call. Returning a pointer type risks some code mutating it accidentally.

Reply:
I'm going to fix this in a follow-up PR.